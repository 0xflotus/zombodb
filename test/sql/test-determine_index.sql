SELECT zdb.determine_index('idxso_posts');
SELECT zdb.determine_index('so_posts');