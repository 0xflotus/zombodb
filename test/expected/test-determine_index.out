SELECT zdb.determine_index('idxso_posts');
 determine_index 
-----------------
 idxso_posts
(1 row)

SELECT zdb.determine_index('so_posts');
 determine_index 
-----------------
 idxso_posts
(1 row)

